lib/cmds/ontologies_cmds/import.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const { put } = require('../../api');
+const print = require('../../print');
+const stdin = require('../../stdin');
+
+const { chain } = require('stream-chain');
+
+const { parser } = require('stream-json');
+const { streamValues } = require('stream-json/streamers/StreamValues');
+
+exports.command = 'import <project>';
+exports.desc = 'Import an ontology';
+exports.builder = yargs => {
+  yargs.positional('project', {
+    describe: 'a project id',
+    type: 'string'
+  });
+};
+
+exports.handler = async argv => {
+  const project = argv.project;
+  const path = `/v1/terminology/projects/${project}/relationships/`;
+
+  const pipeline = chain([
+    stdin(),
+    parser(),
+    streamValues(),
+    async (data) => {
+      const value = data.value;
+      for (let i = 0; i < value.length; i++) {
```

Comment:
Since you're not using the index, seems like you could do a `for (const payload of value)` here.